src/components/Timeline/TimelineActor.tsx

```diff
@@ -0,0 +1,23 @@
+import { Grid } from '@material-ui/core';
+import React from 'react';
+
+import PersonHoverCard from 'components/PersonHoverCard';
+import ZetkinPerson from 'components/ZetkinPerson';
+import { ZetkinUpdate } from 'types/updates';
+
+const TimelineActor: React.FunctionComponent<{
+  actor: ZetkinUpdate['actor'];
+  size: number;
+}> = ({ actor, size }) => {
+  return (
+    <>
+      <Grid item>
+        <PersonHoverCard personId={actor.id}>
+          <ZetkinPerson id={actor.id} name={''} showText={false} size={size} />
+        </PersonHoverCard>
+      </Grid>
```

Comment:
My personal taste is that grid stuff all exists on the same level, and the contents would be their own components. But that's just me. No need to change it. I think it's just clearer then, because then the `<Grid item>` is clearly a child of `<Grid container>`